ocaml/xapi/xha_statefile.ml

```diff
@@ -89,15 +89,54 @@ let assert_sr_can_host_statefile ~__context ~sr ~cluster_stack =
              ]
            )
         )
-  | (_, sm) :: _ ->
+  | (_, sm) :: _ -> (
       if
         (not (List.mem_assoc "VDI_GENERATE_CONFIG" sm.Db_actions.sM_features))
         && not (List.mem_assoc "VDI_ATTACH_OFFLINE" sm.Db_actions.sM_features)
       then
         raise
           (Api_errors.Server_error
              (Api_errors.sr_operation_not_supported, [Ref.string_of sr])
+          ) ;
+      let size = minimum_size in
+      match
+        List.filter
+          (fun self ->
+            true
```

Comment:
What is the `true` doing? I notice it is coming from existing code.

Reply:
I removed it from my suggestion, it's useless

Reply:
it aligns the next 2 clauses by having them both have '&&', I'll drop it.